### DISCUSSION TITLE
0.4.0 release?

### DISCUSSION THREAD

**Discussion by cholmes** [opengeospatial/geoparquet#103] [Administrative]
Hey all, wanted to check in about pushing for a 0.4.0 release. I think we had originally said we'd try to do it by the middle of this week? Like wednesday? I'm happy to cut the release.

I think we're in pretty good shape actually: https://github.com/opengeospatial/geoparquet/milestone/6

There's PR's open for the two main remaining issues: #88 / #92 (z bounds for 3d geometries) and #95 / #96. 

For the first it seemed like maybe there was some lingering discussion about not doing it? Any objections to just merging it in?

For the second it does seem like there's a few little things to maybe clean up, but I'd be for just merging it in at this point and then iterating from there.

The biggest thing it needs is an updated JSON schema file. I think for now we can do it as a separate PR, but I think we should make a PR template that emphasizes that you need to include a JSON file update for any spec update. Just added #102 for that.

I think it'd also be good to get https://github.com/opengeospatial/geoparquet/pull/87 to enable updating of the examples.

**Comment by jorisvandenbossche** [comment]
Yes, sounds good to me. I think the only real blocker is the WKT2 -> PROJJSON change (which seems almost ready to go). And then updating the JSON schema file to follow suite.

The bbox including z or not (https://github.com/opengeospatial/geoparquet/pull/88 / https://github.com/opengeospatial/geoparquet/issues/92) would be nice to resolve, but it requires some more input from others on what they would find useful. But I don't think this one is a blocker for the release.

**Comment by jorisvandenbossche** [comment]
I think we should be ready to go?

**Reply by cholmes** [reply]
Yup! Looks like Kyle is on it with 0.4 changes needed: https://github.com/opengeospatial/geoparquet/pull/104

I'll scan to see if there's any text that needs updated and start a PR if necessary.

**Reply by kylebarron** [reply]
There are a few places in the readme where 0.4 could be updated with 0.5

**Reply by cholmes** [reply]
I think I caught most of them in https://github.com/opengeospatial/geoparquet/pull/105 Though went the route of just stopping trying to guess when we'll do things and just made it all a bit more generic.

**Reply by cholmes** [reply]
@jorisvandenbossche - could you approve https://github.com/opengeospatial/geoparquet/pull/104 and https://github.com/opengeospatial/geoparquet/pull/87 ? 

For #87 I'll also try to run it and try out validation, and then upload the output, so I can do the merge of that one after I check it.

**Reply by kylebarron** [reply]
It might also be good for the python validator to update version number in conjunction with the spec. Otherwise, you have to remember to use `pip install --force-reinstall` because pip won't overwrite an existing package with the same version
